script/Ciel/Day 4/04_01_CIEL04_1.txt

```diff
@@ -67,107 +73,116 @@ She really is in another league.
+"...I'd better head home soon. It's stopped raining, after all."
 }
-[sha:f8dafa52d7a8775094bfb21522c0116cf479f210]{
+[offset:8213]{
 -- Page 8, Offset 8213.
 -- 「あれ、そうなんですか？
-"Oh, is that so?
+"Huh, are you sure?
 }
 [sha:884e4b6c96dbef97938e778e38c81381f64031d0]{
 -- Page 8, Offset 8214.
 -- まだ朝の６時になったばかりですよ、遠野くん」
-It's only six in the morning, Tohno-kun."
+It's only just gone six o'clock you know, Tohno-kun."
 }
 [sha:27262ec2200dd63a62456812ff24221a5ab31dc9]{
 -- Page 9, Offset 8215.
 -- 「そうだけど、昨日は無断外泊しちゃったから。早めに屋敷に戻らないと秋葉になんて言われるか」
-"Yes, but I stayed out yesterday without permission. If I don't get back to the mansion as soon as possible, I don't know what Akiha will say."
+"That may be true, but last night I stayed out without permission.
+ I shudder to think what Akiha will have to say if I don't get back to the mansion pronto."
 }
 [sha:5e74542229e0ca3eb3c4427065af5253fc8eaa0c]{
 -- Page 10, Offset 8216.
 -- 「秋葉……ああ、妹さんのことですね。
-"Akiha...... Oh, you mean your sister.
+"Akiha... Oh, you mean your sister!
 }
 [sha:2d78ae29c4bebc0e873e4aa518fa4d2d33eee692]{
 -- Page 10, Offset 8217. Mods: @n.
 -- 昨日電話しておきましたから、そのあたりのお話は
-I called her yesterday,
+I gave her a call yesterday,
 }
 [sha:2c1be3b828e7afb67448685aeb4646f4099d5bb5]{
 -- Page 10, Offset 8218. Glued.
 -- 大丈夫だと思いますよ」
-so I think you'll be fine."
+ so I think you're clear on that front."
 }
 [sha:f174c758f0197d0435e6f261f51c9f963500eaf5]{
 -- Page 10, Offset 8219.
 -- さらりと、先輩はとんでもない事を口走った。
-She just casually blurted out something outrageous.
+She just offhandedly mentioned something outrageous.
 }
 [sha:8c9fbcc945a172c11d951b94bbfd9d7068b52302]{
 -- Page 11, Offset 8220.
 -- 「……電話したって、先輩が、うちに？」
-"You called...... my family?"
+"Senpai... you called my family?"
 }
 [sha:13c46612f77a6c5eae790c59289c81209467c265]{
 -- Page 12, Offset 8221. Mods: @k, @e.
 -- 「はい。遠野くんがうちに泊まる事を伝えておかないと、ご家族の方たちが心配すると思って。
-"Yes. I thought your family would be worried if I didn't inform them you were staying at my place.
+"Yes. I thought your family would be worried if I didn't let them know you were spending the night at my place.
 }
 [sha:fb0c66e4ce23670627e6d3fd3016ae77f9a79021]{
 -- Page 12, Offset 8222. Glued. Mods: @x.
 -- ご迷惑でした？」
-I hope I didn't cause you any trouble."
+ Should I not have?"
 }
 [sha:71f5cd20678532f21ff35ecd137c0f39bc22cd05]{
 -- Page 12, Offset 8223.
 -- 「いや、それは――――――」
-"No...that's..."
+"Ah, well..."
 }
 [sha:c9090eab33a64c3c8fff72b5d0cb44f9a34757cd]{
 -- Page 13, Offset 8224.
 -- わなわなと震える喉。
-My throat quivered in fear.
+My voice is trembling with fear.
 }
 [sha:2fa4ca37b7c25482dbaf293a28079d556cf2abf7]{
 -- Page 13, Offset 8225.
 -- シエル先輩がうちに電話したっていう事は、つまり、同年代の女生徒が『遠野志貴はうちに泊まります』と、あのお厳しい当主様に伝えたという事だ。
-Then that means, a young woman; one around my age, told our strict family head I'd be staying at her place.
+Ciel-senpai called the mansion.
+ In other words, a female student the same age as me called up our totalitarian family head and told her that
```

Comment:
> told her: 'Tohno

Reply:
I kinda like the inline style? But maybe I am prejudiced against colons

Reply:
my problem is with the "that". If you want to keep the line colon-free, just drop it